features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewScope.kt

```diff
@@ -828,6 +832,11 @@ internal open class RumViewScope(
             memoryVitalMonitor.unregister(memoryVitalListener)
             frameRateVitalMonitor.unregister(frameRateVitalListener)
             networkSettledMetricResolver.viewWasStopped()
+
+            viewEndedMetricDispatcher.sendViewEnded(
```

Comment:
Is it intentional that we are using a different lingvo: `ended` vs `stopped`? Shouldn't we align?

View may be stopped, but still be in the list of the scopes (but not active ones), waiting for the child scopes to finish.

Once all childs scopes finish, the view is considered as `complete` (see https://github.com/DataDog/dd-sdk-android/blob/6c54695c4f36678a840cb45c3016882a90185b90/features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewScope.kt#L1314-L1322).

 Is `ended` here an equivalent of `stopped` or `complete`?

Reply:
Nice catch here - changes followed from this note lead me to found potential issue. 
The `ended` here comes from metric name rather than android sdk notation so I prefer to keep it as it is ( to be aligned with `SessionEndedMetric`), but entire method call now checks for child scopes